bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -345,8 +345,8 @@ spec:
             --ovn-empty-lb-events \
             --loglevel "${OVN_KUBE_LOG_LEVEL}" \
             ${hybrid_overlay_flags} \
-            --metrics-bind-address "0.0.0.0:9102" \
-            --sb-address "{{.OVN_SB_DB_LIST}}" \
+            --metrics-bind-address "0.0.0.0:29102" \
```

Comment:
This should be `127.0.0.1`, right? Or else we don't actually add any security...

Reply:
I thought the point of this was avoiding MiTM, not forbidding random people to check metrics over plain HTTP. 
But I'm fine moving it to 127.0.0.1